src/main/java/seedu/address/model/job/JobNumber.java

```diff
@@ -0,0 +1,39 @@
+package seedu.address.model.job;
+
+//@author owzhenwei
+/**
+ * Represent a job number in the servicing manager
+ */
+public class JobNumber {
+    private static int nextJobNumber;
+
+    public final String jobNumber;
+
+    public JobNumber() {
+        jobNumber = Integer.toString(nextJobNumber);
+        incrementNextJobNumber();
+    }
+
+    /**
+     * Initialize the next job number of the car servicing manager
+     */
+    public static void initNextJobNumber(String args) {
```

Comment:
This class should probably use a singleton design.

For this method name I think `initialize` would sound better than `initNextJobNumber` if its an one-off thing.